### DISCUSSION TITLE
Unable to use Chakra UI 3 style props on custom components.

### DISCUSSION THREAD

**Discussion by DRD161** [chakra-ui/chakra-ui#9360] [Q&A] [answered]
I am migrating to Chakra 3 and I am currently having issues using style props on my custom components. I am attempting to use `mt={4}` on my component but I get an error saying that I need to add `mt` as a prop on the component itself. I have already generated types based on my theme using the CLI.

**Comment by segunadebayo** [comment, marked as answer]
Style props only exist on Chakra elements. Wrap the component:

```tsx
const Card = chakra("div")

function Card(props) {
  return <Box {...props} />
}
```

`chakra(YourComponent)` also works if you forward `className`.
